src/apps/miscapps/generateBaseCellNeighbors.c

```diff
@@ -221,9 +243,52 @@ static void generate() {
 
     auditBaseCellNeighbors(baseCellNeighbors, baseCellRotations);
 
+    time_t rawTime;
+    time(&rawTime);
+    struct tm tmStruct;
+    reentrant_gmtime(&tmStruct, &rawTime);
+    const char licenseFormat[] =
+        "/*\n"
+        " * Copyright 2016-%d Uber Technologies, Inc.\n"
```

Comment:
This seems like a lot of work just to automatically format the date. I don't think the parts of the baseCells.c file unrelated to the baseCellNeighbors/baseCellNeighbor60CCWRots should be generated. (Those two were the only parts of the file that were generated from this - maybe those two could be split off to their own, generated file.)

Could you revert this change, and we can revisit it in another issue or PR?